lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -401,12 +401,24 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
     array_push($signup_takeover_recommended_campaigns, $campaign_array);
   }
 
-  // Send 3 campaigns to signup takeover, prompting user to signup for multiple campaigns.
+  // Check to see if this is a user's first signup for a campaign.
+  $existing_user = dosomething_signup_not_first_signup($user->uid);
+  if ($existing_user) {
+    $user_status = 'repeat-signup-existing';
```

Comment:
I wonder if this should just be more general `existing-user-signup` and then `new-user-signup` cause it doesn't really matter if it is a repeat signup until we get into the js.